pkg/install/install.go

```diff
@@ -253,7 +253,10 @@ func (i *Installer) runSteps(ctx context.Context, steps []interface{}) error {
 				if _, ok := err.(*api.CloudError); !ok {
 					err = fmt.Errorf("%s: %s", runtime.FuncForPC(reflect.ValueOf(step).Pointer()).Name(), err)
 				}
+				i.log.Warnf("ran step but got an error (%s)", err)
 				return err
+			} else {
+				i.log.Infof("ran step %s successfully", runtime.FuncForPC(reflect.ValueOf(step).Pointer()).Name())
```

Comment:
would be good to have this and the log on line 256 to be in the same format (where and how you put the step name)